lib/events.js

```diff
@@ -137,8 +137,8 @@ EventEmitter.init = function(opts) {
     }
     this[kCapture] = Boolean(opts.captureRejections);
   } else {
-    // Assigning it directly a prototype lookup, as it slighly expensive
-    // and it sits in a very sensitive hot path.
+    // Assigning it directly a prototype lookup, as it is slightly 
+    // expensive and it sits in a very sensitive hot path.
```

Comment:
There’s also the question of what `it` refers to and the first half of the sentence also misses a verb … I’m assuming it should be something like `Assigning the kCapture property directly saves an expensive prototype lookup in a very sensitive hot path.`?

Reply:
Resolved in https://github.com/nodejs/node/pull/31026/commits/d568f2232a4a8ca0086b6d5c06e3fbc086a747a5